phalanges:
     - link_geom: some_geom 
       link_frame_id: some_frame_id
       link_name: some_name
       L_T_Cref: [1.1,2.1,3.1,4.5,1.1,2.1,3.1,4.5,1.1,2.1,3.1,4.5]
    
     - link_geom: some_other_geom 
       link_frame_id: some_other_frame_id
       link_name: some_other_name
       L_T_Cref: [1.1,2.1,3.1,4.5,1.1,2.1,3.1,4.5,1.1,2.1,3.1,4.5]

  

  

